pom.xml

```diff
@@ -657,6 +674,12 @@
                     <version>${maven-bundle-plugin-version}</version>
                 </plugin>
 
+                <plugin>
+                    <groupId>org.commonjava.maven.plugins</groupId>
+                    <artifactId>directory-maven-plugin</artifactId>
```

Comment:
When I tried to build only one sub-component of Curator maven-license-plugin always failed as didn't find the file **/src/etc/header.txt**.
This plugin helps to find the root directory of the project and than it can be used in the license plugin as **${main.basedir}/src/etc/header.txt**

Reply:
very good ! thank you very much !